vue-components/stories/Lookup.stories.ts

```diff
@@ -1,53 +1,81 @@
 import Lookup from '@/components/Lookup';
 import { Component } from 'vue';
+import { MenuItem } from '@/components/MenuItem';
 
 export default {
 	component: Lookup,
 	title: 'Lookup',
 };
 
+const vegetableItems = [
+	{
+		label: 'potato',
+		description: 'root vegetable',
+		value: 'Q10998',
+	},
+	{
+		label: 'carrot',
+		description: 'root vegetable, usually orange in color',
+		value: 'Q81',
+	},
+	{
+		label: 'zucchini',
+		description: 'Edible summer squash, typically green in colour',
+		value: 'Q7533',
+	},
+	{
+		label: 'eggplant',
+		description: 'plant species Solanum melongena',
+		value: 'Q7540',
+	},
+	{
+		label: 'broccoli',
+		description: 'edible green plant in the cabbage family',
+		value: 'Q47722',
+	},
+	{
+		label: 'cauliflower',
+		description: 'vegetable, for the plant see Q7537 (Brassica oleracea var. botrytis)',
+		value: 'Q23900272',
+	},
+	{
+		label: 'brussels sprouts',
+		description: 'vegetable',
+		value: 'Q150463',
+	},
+];
+
 export function all(): Component {
 	return {
 		components: { Lookup },
 		data() {
 			return {
-				menuItems: [
-					{
-						label: 'potato',
-						description: 'root vegetable',
-					},
-					{
-						label: 'carrot',
-						description: 'root vegetable, usually orange in color',
-					},
-					{
-						label: 'zucchini',
-						description: 'Edible summer squash, typically green in colour',
-					},
-					{
-						label: 'eggplant',
-						description: 'plant species Solanum melongena',
-					},
-					{
-						label: 'broccoli',
-						description: 'edible green plant in the cabbage family',
-					},
-					{
-						label: 'cauliflower',
-						description: 'vegetable, for the plant see Q7537 (Brassica oleracea var. botrytis)',
-					},
-					{
-						label: 'brussels sprouts',
-						description: 'vegetable',
-					},
-				],
+				search: '',
+				selectedItem: null,
 			};
 		},
+
+		computed: {
+			menuItems(): MenuItem[] {
+				return vegetableItems.filter( ( item ) => item.label.includes( this.search ) );
```

Comment:
I wonder (as I assume we will be getting the items from an external api) whether we should also include loading indication behaviour here. @SaiSan-WMDE did you all discuss this in the QB hike before?

Reply:
Another point to think about is whether the intent was that there would be filtering capabilities after the retrieved items have been cached ( i.e. filter on an already existing list), and if that filtering behaviour doesn't actually belong within the lookup component itself. (I'm sure you probably discussed this before, but I'm not fully caught up on this)